example/README.md

```diff
@@ -0,0 +1 @@
+The test folder provides a very basic usage example. 
```

Comment:
Not sure this is really worth having? but :man_shrugging:

Reply:
Added a proper example.